src/components/data_search/DatasetSearchTable.js

```diff
@@ -30,8 +30,8 @@ const datasetTableHeader = [
   'Data Use',
   'Data Types',
   'Participants',
-  'Data Location',
   'DAC',
```

Comment:
Just checking the ticket, should the 'DAC' header be changed to 'Access Type'?
```suggestion
  'Access Type',
```